src/utils/network.py

```diff
@@ -11,7 +12,7 @@
 
 
 @beartype
-def download_sync(url: str, save_path: str) -> None:
+def download_sync(url: str, save_path: str | Path) -> None:
```

Comment:
also supports pathlib Paths. in future we want to move away from str representation of paths.